x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -17,3 +17,26 @@ file where between(file_path, "dev", ".json", true) == "\\TestLogs\\something"
 [[queries]]
 query = 'process where string(serial_event_id) = "1"'
 expected_event_ids  = [1]
+
+[[queries]]
+expected_event_ids  = [98]
+notes = "regexp doesn't support character classes"
```

Comment:
I don't think you are correct about this statement.
This is a valid query in SQL: `SELECT first_name FROM test_emp WHERE first_name RLIKE 'A[m-o]+.*'` which gets translated into ES query DSL like this:

```
    "query": {
        "regexp": {
            "first_name.keyword": {
                "value": "A[m-o]+.*",
                "flags_value": 65535,
                "max_determinized_states": 10000,
                "boost": 1
            }
        }
    }
```

Which is valid and works.

Reply:
> If you are talking about shorthand character classes, yes, those are not supported by Elasticsearch.

yep. good catch.. updated the comment to be more specific. 521b3f9d137